pipelines/jobs/configurations/jdk16_pipeline_config.groovy

```diff
@@ -103,7 +103,10 @@ class Config16 {
                 buildArgs : [
                         hotspot : '--jvm-variant client,server'
                 ],
-                test                : ['sanity.openjdk']
+                test                : [
+                        nightly: [],
+                        weekly : ['sanity.openjdk', 'sanity.perf', 'sanity.system', 'extended.system']
```

Comment:
You could remove weekly override here completely (though its benign unless we actually had a weekly build pipeline scheduled that would run them or tried to run a release build, which would run both nightly and weekly lists), but given the discussion in Slack, we will plan to address this and other platforms with another issue/PR to enable testing on jdk16

Reply:
sure, I was just trying to make it consistent with x64Windows